From d979cd240d5aae4344c705524d05dbe5792695b4 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <grinberg@compulab.co.il>
Date: Mon, 23 Mar 2015 10:21:26 +0200
Subject: [PATCH 51/59] ARM: i.MX6: dts: add board files for sbc-fx6 DL/S

Add board files for sbc-fx6 DL/S modules and Utilite Value.

Signed-off-by: Valentin Raevsky <valentin@compulab.co.il>
[grinberg@compulab.co.il: fix dtsi files names as per previous patches]
Signed-off-by: Igor Grinberg <grinberg@compulab.co.il>
---
 arch/arm/boot/dts/imx6dl-sbc-fx6.dts  |   23 +++++++++++++++++++++++
 arch/arm/boot/dts/imx6dl-sbc-fx6m.dts |   23 +++++++++++++++++++++++
 2 files changed, 46 insertions(+)
 create mode 100644 arch/arm/boot/dts/imx6dl-sbc-fx6.dts
 create mode 100644 arch/arm/boot/dts/imx6dl-sbc-fx6m.dts

diff --git a/arch/arm/boot/dts/imx6dl-sbc-fx6.dts b/arch/arm/boot/dts/imx6dl-sbc-fx6.dts
new file mode 100644
index 0000000..723b654
--- /dev/null
+++ b/arch/arm/boot/dts/imx6dl-sbc-fx6.dts
@@ -0,0 +1,23 @@
+/*
+* Copyright 2015 CompuLab Ltd.
+*
+* Author: Valentin Raevsky <valentin@compulab.co.il>
+*
+* The code contained herein is licensed under the GNU General Public
+* License. You may obtain a copy of the GNU General Public License
+* Version 2 or later at the following locations:
+*
+* http://www.opensource.org/licenses/gpl-license.html
+* http://www.gnu.org/copyleft/gpl.html
+*/
+
+/dts-v1/;
+#include "imx6dl.dtsi"
+#include "imx6qdl-cm-fx6.dtsi"
+#include "imx6qdl-sb-fx6x.dtsi"
+#include "imx6qdl-sb-fx6.dtsi"
+
+/ {
+	model = "CompuLab CM-FX6 on SBC-FX6";
+	compatible = "compulab,cm-fx6", "compulab,sbc-fx6", "fsl,imx6dl";
+};
diff --git a/arch/arm/boot/dts/imx6dl-sbc-fx6m.dts b/arch/arm/boot/dts/imx6dl-sbc-fx6m.dts
new file mode 100644
index 0000000..f66b177
--- /dev/null
+++ b/arch/arm/boot/dts/imx6dl-sbc-fx6m.dts
@@ -0,0 +1,23 @@
+/*
+* Copyright 2015 CompuLab Ltd.
+*
+* Author: Valentin Raevsky <valentin@compulab.co.il>
+*
+* The code contained herein is licensed under the GNU General Public
+* License. You may obtain a copy of the GNU General Public License
+* Version 2 or later at the following locations:
+*
+* http://www.opensource.org/licenses/gpl-license.html
+* http://www.gnu.org/copyleft/gpl.html
+*/
+
+/dts-v1/;
+#include "imx6dl.dtsi"
+#include "imx6qdl-cm-fx6.dtsi"
+#include "imx6qdl-sb-fx6x.dtsi"
+#include "imx6qdl-sb-fx6m.dtsi"
+
+/ {
+	model = "CompuLab CM-FX6 on SBC-FX6m";	
+	compatible = "compulab,cm-fx6", "compulab,sbc-fx6m", "fsl,imx6dl";
+};
-- 
1.7.9.5

